scripts/schedule.py

```diff
@@ -26,6 +26,9 @@
   -N <num>, --num <num>                Number of times to run/queue the job
                                        [default: 1]
 
+  --first-in-suite                     Mark the first job in a suite so suite
+                                       can note down the rerun releated info
```

Comment:
s/rerun releated/rerun-related/ (typo, plus this is being used as an adjective so it needs a hyphen)